### PR TITLE
Highlight with keyword

### PR DESCRIPTION
**Pull request by inohiro**
Highlight `with` keyword like this. Please 🙏 

### Before

with_before

### After

with_after

### CODE REVIEW

**Comment by C-Bouthoorn** [comment]
Send you a pull request with specs: https://github.com/inohiro/language-sql/pull/1

**Comment by C-Bouthoorn** [comment]
Okay, that was a typo on my side. Let me try again

**Comment by 50Wliu** [comment]
FYI: Specs really aren't needed if you're just adding a keyword to an already-existing list.

**Comment by foresmac** [comment]
:tada:
